src/main/java/org/cryptomator/cryptofs/health/dirid/OrphanDir.java

```diff
@@ -68,28 +67,27 @@ public void fix(Path pathToVault, VaultConfig config, Masterkey masterkey, Crypt
 
 		Path recoveryDir = prepareRecoveryDir(pathToVault, cryptor.fileNameCryptor());
 		if (recoveryDir.toAbsolutePath().equals(orphanedDir.toAbsolutePath())) {
-			// recovery dir was orphaned, already recovered by prepare method
-			return;
-		} else {
-			var stepParentDir = prepareStepParent(dataDir, recoveryDir, cryptor.fileNameCryptor(), orphanDirIdHash);
-
-			try (var orphanedContentStream = Files.newDirectoryStream(orphanedDir)) {
-				AtomicInteger fileCounter = new AtomicInteger(1);
-				AtomicInteger dirCounter = new AtomicInteger(1);
-				AtomicInteger symlinkCounter = new AtomicInteger(1);
-				String longNameSuffix = createClearnameToBeShortened(config.getShorteningThreshold());
+			return; //recovery dir was orphaned, already recovered by prepare method
+		}
 
-				for (Path orphanedResource : orphanedContentStream) {
-					var newClearName = switch (determineCiphertextFileType(orphanedResource)) {
+		var stepParentDir = prepareStepParent(dataDir, recoveryDir, cryptor.fileNameCryptor(), orphanDirIdHash);
+		AtomicInteger fileCounter = new AtomicInteger(1);
+		AtomicInteger dirCounter = new AtomicInteger(1);
+		AtomicInteger symlinkCounter = new AtomicInteger(1);
+		String longNameSuffix = createClearnameToBeShortened(config.getShorteningThreshold());
+		try (var orphanedContentStream = Files.newDirectoryStream(orphanedDir)) {
+			for (Path orphanedResource : orphanedContentStream) {
+				//@formatter:off
```

Comment:
what happens without `@formatter:off`? The switch expression looks pretty normal to me

Reply:
the formatter aligns the closing bracket of the switch expression with the line start. i'd rather want to have it indented. The formatter comments will keep this in subsequent reformats.